exec/exec.go

```diff
@@ -12,12 +12,14 @@ import (
 // dcos-go/exec is a os/exec wrapper. It implements io.Reader and can be used to read both STDOUT and STDERR.
 //
 // Usage:
-// ce := exec.Run("bash", []string{"infinite.sh"}, exec.Timeout(3 * time.Second))
+// ctx, cancel := context.WithTimeout(context.Background(), 3*time.Second)
+// defer cancel()
 //
+// ce, _ := exec.Run(ctx, "bash", []string{"infinite.sh"})
 // io.Copy(os.Stdout, ce)
 // err := <- ce.Done
 // if err != nil {
-// 	log.Error(err)
+// 	log.Fatal(err)
```

Comment:
I assumed this was the log package from the std library? Which doesn't have `Error()`.